.gitmodules

```diff
@@ -0,0 +1,16 @@
+[submodule "mcsema"]
+	path = mcsema
+	url = https://github.com/ekilmer/mcsema.git
+	branch = vcpkg
+[submodule "remill"]
+	path = remill
+	url = https://github.com/ekilmer/remill.git
+	branch = vcpkg
+[submodule "anvill"]
+	path = anvill
+	url = https://github.com/ekilmer/anvill.git
+	branch = vcpkg
+[submodule "rellic"]
+	path = rellic
+	url = https://github.com/ekilmer/rellic.git
+	branch = vcpkg
```

Comment:
The content of these submodules isn't published and is only for tests, but it should probably be changed to the official lifting-bits repos.

The question is should they be changed after or before this branch is merged?